kernel/smp.c

```diff
@@ -984,6 +984,28 @@ static int __init maxcpus(char *str)
 
 early_param("maxcpus", maxcpus);
 
+static cpumask_var_t cpu_boot_mask;
+
+static int __init boot_cpus(char *str)
+{
+	int r;
+	cpumask_var_t mask;
+
+	alloc_bootmem_cpumask_var(&mask);
+	r = cpulist_parse(str, mask);
+	if (r < 0) {
+		pr_warn("Invalid boot_cpus parameter\n");
+		goto ret;
+	}
+	cpu_boot_mask = mask;
+	mask = NULL;
+ret:
+	free_bootmem_cpumask_var(mask);
+	return 1;
+}
+
+__setup("boot_cpus=", boot_cpus);
```

Comment:
Need documentation for it.
Ref: Documentation/admin-guide/kernel-parameters.txt